### DISCUSSION TITLE
Workflow to create translation pull request is set to 4.1-dev

### DISCUSSION THREAD

**Discussion by Valc** [joomla/joomla-cms#38684] [General] [answered]
Hi
Reading about how workflows are working  i have noticed that "create-translation-pull-request-v4.yml" is pointing to the 4.1-dev branch and not to 4.2-dev.

https://github.com/joomla/joomla-cms/blob/0aaf6280042192ec10fee9d3b8cadf197604f3d0/.github/workflows/create-translation-pull-request-v4.yml#L40

https://github.com/joomla/joomla-cms/blob/0aaf6280042192ec10fee9d3b8cadf197604f3d0/.github/workflows/create-translation-pull-request-v4.yml#L75

I imagine that data should be updated to match the current branch.

Regards

**Comment by wojsmol** [comment, marked as answer]
Hi
Pull Request to fix this was created - see https://github.com/joomla/joomla-cms/pull/38698
